noisyatom/blogs/models.py

```diff
@@ -0,0 +1,90 @@
+
+from django.conf import settings
+from django.contrib.contenttypes.models import ContentType
+from django.core.urlresolvers import reverse
+from django.db import models
+from django.db.models.signals import pre_save
+from django.utils import timezone
+from django.utils.safestring import mark_safe
+from django.utils.text import slugify
+
+from markdown_deux import markdown
+
+
+'''
+https://docs.djangoproject.com/en/1.11/ref/models/fields/
+'''
+
+class PostManager(models.Manager):
+
+	def active(self, *args, **kwargs):
```

Comment:
What are *args and **kwargs for? They are not used? What does this post manager do? There are no comments to help the developer know what this is for? There are no doc strings.

Reply:
OK I will put some comments as doc string